articles/app-service-mobile/app-service-mobile-xamarin-forms-get-started-users.md

```diff
@@ -304,17 +342,19 @@ Universal Windows Platform (UWP) projects, but using the **UWP** project (with n
        {
```

Comment:
Fix "WindowsPhone81" and "WinApp" at [here](https://github.com/MicrosoftDocs/azure-docs/pull/2609/commits/b623dae4431091e2f83a32222fbd27829597d0b9?diff=unified#diff-d6c6cdcc45f6207e3316d2bc305c5e9eL295), [here](https://github.com/MicrosoftDocs/azure-docs/pull/2609/commits/b623dae4431091e2f83a32222fbd27829597d0b9?diff=unified#diff-d6c6cdcc45f6207e3316d2bc305c5e9eL301)